examples/eestream/serve-collected/main.go

```diff
@@ -33,6 +33,12 @@ func main() {
 	}
 }
 
+type indexRangerError struct {
```

Comment:
did you know you can actually define types that are only used by one method inside of a method?

Reply:
No. Thanks for the hint!